Assets/Scripts/MotionState.cs

```diff
@@ -32,7 +50,22 @@ public void OnPhotonSerializeView(PhotonStream stream, PhotonMessageInfo info)
             mVelY = (float)stream.ReceiveNext();
             mGrounded = (bool)stream.ReceiveNext();
             mFacingRight = (bool)stream.ReceiveNext();
-            transform.position = (Vector3)stream.ReceiveNext();
+            mPreviousPosition = mAccuratePosition;
+            mAccuratePosition = (Vector3)stream.ReceiveNext();
+
+            mPositionLerpTime = 0;
+            ++mNumUpdates;
+            mPositionUpdateRate = mTotalTime / mNumUpdates;
+        }
+    }
+
+    void Update()
+    {
+        if (!photonView.isMine)
+        {
+            mTotalTime += Time.deltaTime;
+            mPositionLerpTime += Time.deltaTime;
+            transform.position = Vector3.Lerp(mPreviousPosition, mAccuratePosition, mPositionLerpTime / mPositionUpdateRate);

```

Comment:
apologies for missing these comments. Not sure how I didn't see them.
This was resolved offline